core/controllers/story_viewer.py

```diff
@@ -141,7 +141,27 @@ def _record_node_completion(
         completed_node_ids: List[str],
         ordered_nodes: List[story_domain.StoryNode]
     ) -> Tuple[List[str], Optional[str], List[str]]:
-        """Records node completion."""
+        """Records node completion.
+
+        Args:
+            story_id: str. The story ID.
+            node_id: str. The node ID.
+            completed_node_ids: List[str]. A list of IDS of completed nodes.
+            ordered_nodes: List[story_domain.StoryNode]. A list of story
+                nodes in order.
+
+        Returns:
+            3-Tuple(next_exp_ids, next_node_id, completed_node_ids). Where:
+                next_exp_ids: List[str]. The next_exp_ids list contains
+                    the exploration IDs of the next node(s) to be explored
+                next_node_id: str. The ID of the next node to be explored.
+                completed_node_ids: List[str]. An updated of IDS that represents
```

Comment:
@seanlip Sorry I don't understand.   When I ran linters for the file nothing came up.

Reply:
@Helper2020, I think he referred to grammatical mistake in the sentence, "An updated of IDS" doesn't make sense, I think you meant "An updated list of IDs" here ?

Reply:
Ah sorry, @Ash-2k3 is right. I meant "scan" as in https://english.stackexchange.com/questions/66487/meaning-of-scan-in-that-doesnt-scan-in-english (but it might not be a widely-used thing).

Reply:
Done.